OrbitBase/include/OrbitBase/Platform/Windows/Profiling.h

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) 2020 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef ORBIT_BASE_PLATFORM_WINDOWS_PROFILING_H_
+#define ORBIT_BASE_PLATFORM_WINDOWS_PROFILING_H_
+
+#include <Windows.h>
```

Comment:
I would rather we had one header and 2 implementations.. for inlined functions I suppose it is ok to use ifdefs in this file.